mne_qt_browser/_pg_figure.py

```diff
@@ -21,18 +21,26 @@
 from os.path import getsize
 
 import numpy as np
-from qtpy.QtCore import (
-    QEvent,
-    QThread,
-    Qt,
-    Signal,
-    QRectF,
-    QLineF,
-    QPointF,
-    QPoint,
-    QSettings,
-    QSignalBlocker,
-)
+
+try:
+    from qtpy.QtCore import (
+        QEvent,
+        QThread,
+        Qt,
+        Signal,
+        QRectF,
+        QLineF,
+        QPointF,
+        QPoint,
+        QSettings,
```

Comment:
I would also just nest one of these imports, then the rest can be as they were before. It'll make the `diff` smaller as well

Reply:
Thanks for the comments @larsoner